docs/send-data/hosted-collectors/http-source/zoom.md

```diff
@@ -0,0 +1,85 @@
+---
+id: zoom
+title: Zoom Source
+description: An HTTP Zoom Source is an endpoint for receiving traces.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/saas-cloud/zoom.png')} alt="Thumbnail icon" width="50"/><br/>
+
+Zoom HTTP Source is an endpoint for receiving Webhook events that are grouped into the following core event types:
+- Meeting Events
+- Webinar Events
+- Recording Events
+- Zoom Room Events
+- User Events
+- Account Events
+
+For more information on Zoom Webhook events, see this [Zoom web page](https://marketplace.zoom.us/docs/api-reference/webhook-reference).
+
+This section shows you how to configure Webhooks to collect events from Zoom. For more information, see [Create a Webhook-Only App](https://marketplace.zoom.us/docs/guides/getting-started/app-types/create-webhook-only-app).
+
+## Prerequisites
+
+When you create a Zoom Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](https://help.sumologic.com/docs/send-data/hosted-collectors/configure-hosted-collector/).
```

Comment:
We don't use absolute links in docs @JV0812 .

Reply:
fixed.